ember-file-upload/src/test-support.ts

```diff
@@ -24,12 +24,15 @@ import { assert } from '@ember/debug';
   Returns `Promise<void>` which resolves when the application is settled.
 
   @function selectFiles
-  @param {String} selector DOM selector for a FileUpload input
+  @param {(String | HTMLElement)} selector DOM selector for a FileUpload input or the FileUpload input HTML Element
```

Comment:
A few notes:
- `string` type should be lowercase
- Maybe we should rename `selector` to `target`. This is the param name [used by ember's test helpers](https://github.com/emberjs/ember-test-helpers/blob/master/API.md#click). I think it would make the code itself read better, too

```suggestion
  @param {(string | HTMLElement)} target The element or selector representing a file input
```